doc/overview.rst

```diff
@@ -123,8 +124,8 @@ A typical Dune project will have a ``dune-project`` and one or more
 interesting things are: libraries, executables, tests, documents to install,
 etc.
 
-We recommended organizing your project to have exactly one library
-per directory. You can have several executables in the same directory, as long
-as they share the same build configuration. If you'd like to have multiple
+We recommended organizing your project to have exactly one library per
```

Comment:
```suggestion
We recommended organising your project to have exactly one library per
```
Same with the British spelling. (I think in previous docs we did this, so if it should be American, I'll make a note of it.)